app/controllers/api/v1/forgot_passwords_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class ForgotPasswordsController < ApplicationController
+      skip_before_action :doorkeeper_authorize!
+
+      before_action :set_resource
+
+      def create
```

Comment:
@hoangmirs I think we might not need controller at all, can we try using endpoint already provided by devise?

Reply:
Thank you, I change to inherit from PasswordController of Devise and just change a bit to response JSON data